Common/DataModel/TrackSelectionTables.h

```diff
@@ -19,12 +19,63 @@ namespace o2::aod
 namespace track
 {
 // Columns to store the DCA to the primary vertex
-DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //!
-DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //!
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //! Impact parameter in XY of the track to the primary vertex
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //! Impact parameter in Z of the track to the primary vertex
+
+struct TrackSelectionFlags {
+ public:
+  typedef uint16_t flagtype;
```

Comment:
@njacazio here a simple curiosity: do you define a new `typedef` just to be flexible in case we need in the future to extend this `uint16_t` to a bigger data format, eg `uint32_t` oreven `uint64_t` in case we need to insert new cuts in the bitmap? Was it the idea? If yes, fine for me.

Reply:
exactly, this was my point